src/main/java/edu/kis/powp/jobs2d/drivers/usageMonitor/IDriverMonitor.java

```diff
@@ -0,0 +1,9 @@
+package edu.kis.powp.jobs2d.drivers.usageMonitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public interface IDriverMonitor extends Job2dDriver
```

Comment:
It is more than a monitor, it is also a driver.

Reply:
Name changed to IMonitorDriverDecorator.